src/__snapshots__/changelog.spec.js.snap

```diff
@@ -41,29 +49,9 @@ exports[`Changelog > getCommitInfos > parse commits with different tags 1`] = `
     "tags": undefined,
   },
   {
-    "categories": [
-      ":rocket: New Feature",
-    ],
     "commitSHA": "a0000003",
     "date": "2017-01-01",
-    "githubIssue": {
```

Comment:
why is this snapshot changing? 🤔 are we missing the new setup for the issuenumber in question in the test that is generating this snapshot?

Reply:
thats because the PR is also in a0000004, and only 1 PR link will be set